CONTRIBUTING.md

```diff
@@ -6,6 +6,7 @@
     - Verify that the perplexity and the performance are not affected negatively by your changes (use `llama-perplexity` and `llama-bench`)
     - If you modified the `ggml` source, run the `test-backend-ops` tool to check whether different backend implementations of the `ggml` operators produce consistent results (this requires access to at least two different `ggml` backends)
     - If you modified a `ggml` operator or added a new one, add the corresponding test cases to `test-backend-ops`
+- Do not add multiple, unrelated changes to a single PR. It is generally easier for maintainers to review small PRs with a single feature, especially since not every maintainer is familiar with all parts of the codebase. For example, instead of a single PR that fixes a bug and adds support for some model, make separate PRs for the bugfix and model support
```

Comment:
Btw, I think that shorter guidelines are easier to digest and likely improve the rate at which they are followed. For example this one can be phrased like this:

- Create separate PRs for each feature or fix. Avoid combining unrelated changes in a single PR